### PR TITLE
Feature/alp 2/main store

### PR DESCRIPTION
**Pull request by PAKovalev**
Main store refactored.

### CODE REVIEW

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/app/src/scenes/collaboration/pages/LiveStreamPage/LiveStreamPage.tsx [hunk 1]
```diff
@@ -37,8 +36,8 @@ const LiveStreamPage: FC = () => {
         subtitle={t('liveStream.subtitle')}
         isAdmin={spaceStore.isAdmin}
         spaceId={space.id}
-        isSpaceFavorite={favoriteStore.isFavorite(space.id)}
-        toggleIsSpaceFavorite={favoriteStore.toggleFavorite}
+        isSpaceFavorite={false}
+        toggleIsSpaceFavorite={() => {}}
```
Comment (FIZLIN) [inline packages/app/src/scenes/collaboration/pages/LiveStreamPage/LiveStreamPage.tsx:40]: We're ditching the favorite logic for now?

File: packages/app/src/scenes/collaboration/pages/StageModePage/components/templates/StageModeGuest/StageModeGuest.tsx [hunk 2]
```diff
@@ -59,13 +58,13 @@ const StageModeGuest: React.FC<PropsInterface> = ({onLeaveMeeting}) => {
       <SpaceTopBar
         title={spaceStore.space?.name ?? ''}
         subtitle={t('labels.stageMode')}
-        isSpaceFavorite={favoriteStore.isFavorite(spaceStore.id || '')}
+        isSpaceFavorite={false}
+        toggleIsSpaceFavorite={() => {}}
```
Comment (FIZLIN) [inline packages/app/src/scenes/collaboration/pages/StageModePage/components/templates/StageModeGuest/StageModeGuest.tsx:62]: ditto (_We're ditching the favorite logic for now?_)

**Review by FIZLIN — COMMENTED** [review]
LGTM

**Review by FIZLIN — COMMENTED** [review]
LGTM

**Pull request merged**